app/assets/javascripts/thyme/key_shortcuts.js

```diff
@@ -0,0 +1,112 @@
+/**
+  All key shortcuts should be bundled here.
+*/
+const thymeKeyShortcuts = {
+  /*
+    SHORTCUT LIST:
+      Arrow right - plus ten seconds
+      Arrow left - minus ten seconds
+      f - fullscreen
+      Page up - volume up
+      Page down - volume down
+      m - mute
+  */
+  addGeneralShortcuts: function() {
+    const fullScreenButton = document.getElementById('full-screen');
+    const minusTenButton = document.getElementById('minus-ten');
+    const muteButton = document.getElementById('mute');
+    const plusTenButton = document.getElementById('plus-ten');
+    const video = document.getElementById('video');
+
+    window.addEventListener('keydown', function(evt) {
+      if (thymeAttributes.lockKeyListeners === true) {
+        return;
+      }
+      const key = evt.key;
+      if (key === ' ') {
+        if (video.paused === true) {
+          video.play();
+        } else {
+          video.pause();
+        }
+      } else if (key === 'ArrowRight') {
+        $(plusTenButton).trigger('click');
+      } else if (key === 'ArrowLeft') {
+        $(minusTenButton).trigger('click');
+      } else if (key === 'f') {
+        $(fullScreenButton).trigger('click');
+      } else if (key === 'm') {
+        $(muteButton).trigger('click');
+      } else if (key === 'PageUp') {
+        video.volume = Math.min(video.volume + 0.1, 1);
+      } else if (key === 'PageDown') {
+        video.volume = Math.max(video.volume - 0.1, 0);
+      }
+    });
+  },
+
+  /*
+    Thyme player specific
+
+    SHORTCUT LIST:
+      Arrow Up - next chapter
+      Arrow Down - previous chapter
+      i - toggle interactive area
+  */
+  addPlayerShortcuts() {
+    const iaButton = document.getElementById('ia-active');
+    const nextChapterButton = document.getElementById('next-chapter');
+    const previousChapterButton = document.getElementById('previous-chapter');
+
+    window.addEventListener('keydown', function(evt) {
+      if (thymeAttributes.lockKeyListeners === true) {
```

Comment:
get rid of `=== true` (multiple occurrences throughout this PR) if you can be sure that the underlying variable can only hold `boolean` values.

Aside: To test for non-existence use `if (!myProperty)` but make sure you don't need the distinction between `undefined` and `null`. E.g. search for `=== null` in this PR to apply this change.

Reply:
Ok, I changed this everywhere. I have to admit, that I tend to use the `==` operator to check if an object is null rather than `===` for having the `undefined` case included (maybe even in some situation where it can be ensured that this doesn't happen). Still, for now I only replaced the appearances of `=== null`.